md/ts-jsdoc.md

```diff
@@ -44,17 +44,48 @@ When installing a dependency from a git url (ie. PRs depending on other PRs) the
 ```
 > `yarn` needs a .npmignore file to properly install dependencies with `prepare` scripts that create extra files that need to be packed in.
 
-## Adding types with JSDoc
-
-Typescript can infere lots of the types without any help, but you can improve your code types by using just JSDoc for that follow the official TS documentation https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html. 
-
 ## Must read references
 [10 Insights from Adopting TypeScript at Scale](https://www.techatbloomberg.com/blog/10-insights-adopting-typescript-at-scale/)  
 [Typescript official performance notes](https://github.com/microsoft/TypeScript/wiki/Performance)  
 [TypeScript: Don’t Export const enums](https://ncjamieson.com/dont-export-const-enums/)  
 [TypeScript: Prefer Interfaces](https://ncjamieson.com/prefer-interfaces/)
 
 
+## Adding types with JSDoc
+
+Typescript can infere lots of the types without any help, but you can improve your code types by using just JSDoc for that follow the official TS documentation https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html. 
+
+## Manage dependencies types
+When dependencies don't publish types you have two options.
+
+### NPM from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)
+```bash
+npm install @types/tape
+```
+
+### Vendor type declarations
+Create a `types` folder at the root of your project to keep all your vendored types. Inside create one folder per dependency using the dependency name as the folder name and inside a create `index.d.ts` file with the types. 
```

Comment:
Does that work without needing to configure anything in package.json ? Specifically since `tsconfig.json` is not going to be in `dist` how will TS know to look into `dist/types` ?

Reply:
ts will just follow imports in `dist/src` to `dist/types`